### PR TITLE
Fix Blob.Created trigger to only handle Blob.Created events

### PR DESCRIPTION
**Pull request by marrobi**
Related to #3795

Update the `airlock_processor/BlobCreatedTrigger/function.json` to remove the binding for Blob.Deleted events.

* Remove the `dataDeletionEvent` binding from the `airlock_processor/BlobCreatedTrigger/function.json` file.
* Update the `CHANGELOG.md` to include the bug fix for the issue "Delete of in-progress container is triggered twice" (#4035).

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/microsoft/AzureTRE/issues/3795?shareId=c4a5a55b-a1cf-4ddf-9c9e-170d825680c1).

### CODE REVIEW

**Comment by marrobi** [comment]
Increment version

* Update version from 0.7.1 to 0.7.2

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/microsoft/AzureTRE/issues/3795?shareId=XXXX-XXXX-XXXX-XXXX).

File: CHANGELOG.md [hunk 1]
```diff
@@ -687,7 +688,7 @@ ENHANCEMENTS:
 * Update docker base images to bullseye ([#2946](https://github.com/microsoft/AzureTRE/pull/2946)
 * Support updating the firewall when installing via makefile/CICD ([#2942](https://github.com/microsoft/AzureTRE/pull/2942))
 * Add the ability for workspace services to request additional address spaces from a workspace ([#2902](https://github.com/microsoft/AzureTRE/pull/2902))
-* Airlock processor function and api app service work with http2
+* Airlock function internal storage is accessed with private endpoints ([#2980](https://github.com/microsoft/AzureTRE/pull/2980))
```
Comment (marrobi) [inline CHANGELOG.md:691]:
```suggestion
* Airlock processor function and api app service work with http2
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ ENHANCEMENTS:
 BUG FIXES:
 * Fix numbering issue within `bug_report.md` template ([#4028](https://github.com/microsoft/AzureTRE/pull/4028))
 * Disable public network access to the API App Service ([#3986](https://github.com/microsoft/AzureTRE/issues/3986))
+* Delete of in-progress container is triggered twice ([#4035](https://github.com/microsoft/AzureTRE/issues/4035))
```
Comment (tim-allen-ck) [inline CHANGELOG.md:17]: Think this is the wrong issue number?

Comment (marrobi) [reply CHANGELOG.md:17]: I blame AI.

```suggestion
* Delete of in-progress container is triggered twice ([#3795](https://github.com/microsoft/AzureTRE/issues/3795))
```

**Review by tim-allen-ck — APPROVED** [review]

**Comment by marrobi** [comment]
@tim-allen-ck  looks liek a bug in this PR, sorry.

Getting:

Result: Failure Exception: FunctionLoadError: cannot load the BlobCreatedTrigger function: the following parameters are declared in Python but not in function.json: {'dataDeletionEvent'} Stack: File "/azure-functions-

**Comment by marrobi** [comment]
Suggest we revert for the next release, then dig deeper.

**Pull request merged**